facebook/OpenBIC
# fby35: ji: Support bic and vr pldm fwupdate

Summary:
- Support BIC pldm fw update
- Support VR devices(mpq8746,mp2898,mp2894) pldm fw update

Test Plan:
- Buiild code: PASS
- Update BIC: PASS
- Update 3 vr: PASS

Log:
- BMC console:
```
root@bmc-oob:~# fw-util slot1 --update bic Y35BJI.pldm
PLDM image detected.
PLDM fw package info, Platform: Yosemite V3.5
PLDM fw package info, board_id: Java Island(1)
PLDM fw package info, Stage: EVT(1)
PLDM fw package info, component_id: 2
PLDM fw package info, vendor: ast1030(1)
PLDM Package info check passed.
RequestUpdate Success.
PassComponentTable Success.
UpdateComponent Success.
Download offset : 0x00044d80/0x00044df8, size : 0x00000078
TransferComplete.
VerifyComplete.
ApplyComplete.
ActivateFirmwareComplete.
Upgrade of slot1 : bic succeeded
```
